codechecker_common/cli.py

```diff
@@ -57,14 +57,25 @@ def get_data_files_dir_path():
     # If it's not an internal package (pypi package), try to find the data
     # directory.
     import sysconfig
```

Comment:
Self-installed packages under v-envs and such might also need this functionality. How about the following comment making it a bit more clear?

```suggestion
    # If this is a pip-installed package, try to find the data directory.
    import sysconfig
```